api/base/types.go

```diff
@@ -32,7 +32,7 @@ type ModelStatus struct {
 	TotalMachineCount  int
 	CoreCount          int
 	HostedMachineCount int
-	ServiceCount       int
+	ApplicationCount   int
```

Comment:
Considering that this changes the name in the returning structure, I think that the implications are further reaching - facades need to be bumped, transformation code at presentation layer changed, etc...